tests/features/paragraphs/latest_blog_posts_paragraphs.feature

```diff
@@ -0,0 +1,61 @@
+@openy @api
+Feature: Latest Blogs Paragraphs
+  Check that Latest Blog Posts paragraphs output other nodes based on some rules
+
+  Background: Log in
+    Given I am logged in as a user with the "Administrator" role
+
+  Scenario: Paste latest blog post
+    And I go to "/node/add/landing_page"
+    And I fill in "Title" with "Landing"
+    And I select "One Column" from "Layout"
+    When I press "Add Latest blog posts"
```

Comment:
We should generate blog posts in Background job. Here you use demo content, that is unacceptable.